src/Agent.Listener/Program.cs

```diff
@@ -10,15 +11,16 @@ public static class Program
     {
         private static TraceSource s_trace;
         
```

Comment:
Other than minor style compliance, this file was impacted because I removed the CancellationTokenSource from the HostContext object. This was a lingering TODO in the code, and at this point the solution now seemed clear how to do it.